### PR TITLE
Camera tweaks

### PR DESCRIPTION
**Pull request by scruben**

### CODE REVIEW

File: containers/row.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```
Comment (arol) [inline containers/row.js]: It's better to name it after something more explicit. `UserRow` ?

File: components/grid.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    textAlign: 'right'
+  },
+  photo: {
+    height: 350,
+    width: 350,
+    borderRadius: 5,
+  },
+});
+
+const Grid = (props) => (
```
Comment (arol) [inline components/grid.js]: `GridItem`. But if you look in it, it's doing direct references to a challenge. `ChallengeGridItem` ?

File: containers/challengesscreen.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    marginTop: 70
+  }
+});
+
+class ChallengesScreen extends React.Component {
+
+  challengesList = [
```
Comment (arol) [inline containers/challengesscreen.js]: This mock is not the same as the agreed on the api documentation. Use api mocks, and if you're missing something ask to the api responsible to change it.

File: containers/selectfriends.js [hunk 4]
```diff
@@ -14,36 +14,68 @@ import {
...
-    super(props);
 
-    const ds = new ListView.DataSource({rowHasChanged: (r1, r2) => r1 !== r2});
-    this.state = {
-      actual: 3,
-      dataSource: ds.cloneWithRows([
-      {
+class SelectFriends extends React.Component {
+  friendsList = [
+    {
```
Comment (arol) [inline containers/selectfriends.js:28]: This mock is not close either to the api response message. In this case you can't ask facebook to change it 🤓

File: containers/camera.js [hunk 5]
```diff
@@ -50,7 +50,7 @@ class CameraComponent extends Component {
   }
 
   componentWillReceiveProps (nextProps) {
-    if (this.props.isRecording !== nextProps.isRecording && nextProps.token !== '') {
+    if (nextProps.isRecording === false && nextProps.recordedFile !== '') {
```
Comment (arol) [inline containers/camera.js:53]: No changes requested: actually you can check if it has changed by doing `this.props.isRecording !== nextProps.isRecording`.

**Review by arol — CHANGES_REQUESTED** [review]
Please, also resolve conflicts before the reviewer comes into the PR

**Comment by arol** [comment]
Depends on #5

**Review by mapageKA21 — APPROVED** [review]
👍

**Pull request merged**